### ISSUE TITLE
Migration Test in app example fails

### ISSUE THREAD

**Issue by MateusAmin** [simolus3/moor#1954]
Migration Test in app example fails from latest develop.

```sh
$ flutter test
00:01 +0: ... /home/mea/drift/examples/app/test/database_test.dart             test/generated_migrations/schema_v2.dart:10:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v2.dart:15:39: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> name = GeneratedColumn<String?>(
                                      ^
test/generated_migrations/schema_v2.dart:18:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> color = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v2.dart:48:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v2.dart:53:39: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> description = GeneratedColumn<String?>(
                                      ^
test/generated_migrations/schema_v2.dart:56:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> category = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v2.dart:59:41: Error: Type argument 'DateTime?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'DateTime' is from 'dart:core'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<DateTime?> dueDate = GeneratedColumn<DateTime?>(
                                        ^
test/generated_migrations/schema_v1.dart:10:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v1.dart:15:39: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> name = GeneratedColumn<String?>(
                                      ^
test/generated_migrations/schema_v1.dart:18:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> color = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v1.dart:48:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v1.dart:53:39: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> description = GeneratedColumn<String?>(
                                      ^
test/generated_migrations/schema_v1.dart:56:36: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> category = GeneratedColumn<int?>(
                                   ^
test/generated_migrations/schema_v2.dart:12:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(),
                  ^^^^^^^
test/generated_migrations/schema_v2.dart:10:41: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                        ^
test/generated_migrations/schema_v2.dart:17:19: Error: Couldn't find constructor
'StringType'.
      type: const StringType(), requiredDuringInsert: true);
                  ^^^^^^^^^^
test/generated_migrations/schema_v2.dart:15:46: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> name = GeneratedColumn<String?>(
                                             ^
test/generated_migrations/schema_v2.dart:20:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(), requiredDuringInsert: true);
                  ^^^^^^^
test/generated_migrations/schema_v2.dart:18:44: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> color = GeneratedColumn<int?>(
                                           ^
test/generated_migrations/schema_v2.dart:22:42: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, name, color];
                                         ^
test/generated_migrations/schema_v2.dart:22:46: Error: A value of type
'GeneratedColumn<String?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'String?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, name, color];
                                             ^
test/generated_migrations/schema_v2.dart:22:52: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, name, color];
                                                   ^
test/generated_migrations/schema_v2.dart:28:44: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  Set<GeneratedColumn> get $primaryKey => {id};
                                           ^
test/generated_migrations/schema_v2.dart:50:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(),
                  ^^^^^^^
test/generated_migrations/schema_v2.dart:48:41: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                        ^
test/generated_migrations/schema_v2.dart:55:19: Error: Couldn't find constructor
'StringType'.
      type: const StringType(), requiredDuringInsert: true);
                  ^^^^^^^^^^
test/generated_migrations/schema_v2.dart:53:53: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> description = GeneratedColumn<String?>(
                                                    ^
test/generated_migrations/schema_v2.dart:58:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(), requiredDuringInsert: false);
                  ^^^^^^^
test/generated_migrations/schema_v2.dart:56:47: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> category = GeneratedColumn<int?>(
                                              ^
test/generated_migrations/schema_v2.dart:61:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(), requiredDuringInsert: false);
                  ^^^^^^^
test/generated_migrations/schema_v2.dart:59:51: Error: Type argument 'DateTime?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'DateTime' is from 'dart:core'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<DateTime?> dueDate = GeneratedColumn<DateTime?>(
                                                  ^
test/generated_migrations/schema_v2.dart:63:42: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category, dueDate];
                                         ^
test/generated_migrations/schema_v2.dart:63:46: Error: A value of type
'GeneratedColumn<String?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'String?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category, dueDate];
                                             ^
test/generated_migrations/schema_v2.dart:63:59: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category, dueDate];
                                                          ^
test/generated_migrations/schema_v2.dart:63:69: Error: A value of type
'GeneratedColumn<DateTime?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'DateTime?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'DateTime' is from 'dart:core'.
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category, dueDate];
                                                                    ^
test/generated_migrations/schema_v2.dart:69:44: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  Set<GeneratedColumn> get $primaryKey => {id};
                                           ^
test/generated_migrations/schema_v2.dart:115:41: Error: Undefined name
'SqlTypeSystem'.
  DatabaseAtV2(QueryExecutor e) : super(SqlTypeSystem.defaultInstance, e);
                                        ^^^^^^^^^^^^^
test/generated_migrations/schema_v2.dart:115:40: Error: Too many positional
arguments: 1 allowed, but 2 found.
Try removing the extra positional arguments.
  DatabaseAtV2(QueryExecutor e) : super(SqlTypeSystem.defaultInstance, e);
                                       ^
test/generated_migrations/schema_v1.dart:12:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(),
                  ^^^^^^^
test/generated_migrations/schema_v1.dart:10:41: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                        ^
test/generated_migrations/schema_v1.dart:17:19: Error: Couldn't find constructor
'StringType'.
      type: const StringType(), requiredDuringInsert: true);
                  ^^^^^^^^^^
test/generated_migrations/schema_v1.dart:15:46: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> name = GeneratedColumn<String?>(
                                             ^
test/generated_migrations/schema_v1.dart:20:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(), requiredDuringInsert: true);
                  ^^^^^^^
test/generated_migrations/schema_v1.dart:18:44: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> color = GeneratedColumn<int?>(
                                           ^
test/generated_migrations/schema_v1.dart:22:42: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, name, color];
                                         ^
test/generated_migrations/schema_v1.dart:22:46: Error: A value of type
'GeneratedColumn<String?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'String?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, name, color];
                                             ^
test/generated_migrations/schema_v1.dart:22:52: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, name, color];
                                                   ^
test/generated_migrations/schema_v1.dart:28:44: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  Set<GeneratedColumn> get $primaryKey => {id};
                                           ^
test/generated_migrations/schema_v1.dart:50:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(),
                  ^^^^^^^
test/generated_migrations/schema_v1.dart:48:41: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> id = GeneratedColumn<int?>(
                                        ^
test/generated_migrations/schema_v1.dart:55:19: Error: Couldn't find constructor
'StringType'.
      type: const StringType(), requiredDuringInsert: true);
                  ^^^^^^^^^^
test/generated_migrations/schema_v1.dart:53:53: Error: Type argument 'String?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<String?> description = GeneratedColumn<String?>(
                                                    ^
test/generated_migrations/schema_v1.dart:58:19: Error: Couldn't find constructor
'IntType'.
      type: const IntType(), requiredDuringInsert: false);
                  ^^^^^^^
test/generated_migrations/schema_v1.dart:56:47: Error: Type argument 'int?'
doesn't conform to the bound 'Object' of the type variable 'T' on
'GeneratedColumn'.
 - 'Object' is from 'dart:core'.
Try changing type arguments so that they conform to the bounds.
  late final GeneratedColumn<int?> category = GeneratedColumn<int?>(
                                              ^
test/generated_migrations/schema_v1.dart:60:42: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category];
                                         ^
test/generated_migrations/schema_v1.dart:60:46: Error: A value of type
'GeneratedColumn<String?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'String?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category];
                                             ^
test/generated_migrations/schema_v1.dart:60:59: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  List<GeneratedColumn> get $columns => [id, description, category];
                                                          ^
test/generated_migrations/schema_v1.dart:66:44: Error: A value of type
'GeneratedColumn<int?>' can't be assigned to a variable of type
'GeneratedColumn<Object>' because 'int?' is nullable and 'Object' isn't.
 - 'GeneratedColumn' is from
 'package:drift/src/runtime/query_builder/query_builder.dart'
 ('../../drift/lib/src/runtime/query_builder/query_builder.dart').
 - 'Object' is from 'dart:core'.
  Set<GeneratedColumn> get $primaryKey => {id};
                                           ^
test/generated_migrations/schema_v1.dart:112:41: Error: Undefined name
'SqlTypeSystem'.
  DatabaseAtV1(QueryExecutor e) : super(SqlTypeSystem.defaultInstance, e);
                                        ^^^^^^^^^^^^^
test/generated_migrations/schema_v1.dart:112:40: Error: Too many positional
arguments: 1 allowed, but 2 found.
Try removing the extra positional arguments.
  DatabaseAtV1(QueryExecutor e) : super(SqlTypeSystem.defaultInstance, e);
                                       ^
00:01 +1 -1: loading /home/mea/drift/examples/app/test/migration_test.dart [E] 
  Failed to load "/home/mea/drift/examples/app/test/migration_test.dart": Compilation failed for testPath=/home/mea/drift/examples/app/test/migration_test.dart
00:01 +1 -1: Some tests failed.                           
```
On ubuntu with flutter installed as a snap tracking latest/edge/core20 (otherwise this plugin fails as the default snap branch does not have the latest cmake).

```sh
snap info flutter
name:      flutter
summary:   Flutter SDK
publisher: Flutter Team✓
store-url: https://snapcraft.io/flutter
contact:   https://github.com/flutter/flutter/issues
license:   unset
description: |
  Flutter is Google’s UI toolkit for building beautiful, natively compiled
  applications for mobile, web, and desktop from a single codebase.
commands:
  - flutter.dart
  - flutter
  - flutter.openurl
snap-id:      YO0Adf4fqgxL7i3SaCn00oxEfd5CNQ63
tracking:     latest/edge/core20
refresh-date: 4 days ago, at 10:45 EDT
channels:
  latest/stable:    0+git.1586a01 2022-05-24 (126) 204MB classic
  latest/candidate: ↑                                    
  latest/beta:      ↑                                    
  latest/edge:      0+git.1586a01 2022-05-21 (126) 204MB classic
installed:          0+git.5885851            (127) 215MB classic
```

**Comment by MateusAmin** [comment]
Tried to rerun the migration dart file generation and got an error.

```sh
$ flutter pub run build_runner build --delete-conflicting-outputs
[INFO] Generating build script...
[INFO] Generating build script completed, took 340ms

[INFO] Precompiling build script......
[INFO] Precompiling build script... completed, took 5.6s

[INFO] Initializing inputs
[INFO] Building new asset graph...
[INFO] Building new asset graph completed, took 1.9s

[INFO] Checking for unexpected pre-existing outputs....
[INFO] Deleting 1 declared outputs which already existed on disk.
[INFO] Checking for unexpected pre-existing outputs. completed, took 2ms

[INFO] Running build...
[INFO] Generating SDK summary...
[INFO] 1.7s elapsed, 1/16 actions completed.
[INFO] 3.0s elapsed, 1/16 actions completed.
[INFO] Generating SDK summary completed, took 3.5s

[INFO] 4.7s elapsed, 1/16 actions completed.
[INFO] 5.8s elapsed, 1/16 actions completed.
[INFO] 6.9s elapsed, 1/16 actions completed.
[INFO] 11.7s elapsed, 1/16 actions completed.
[INFO] 14.4s elapsed, 1/16 actions completed.
[INFO] 15.4s elapsed, 396/413 actions completed.
[INFO] 16.5s elapsed, 1073/1084 actions completed.
[INFO] 17.6s elapsed, 1528/1532 actions completed.
[INFO] build_web_compilers:entrypoint on web/worker.dart:Running `dart compile js` with --enable-asserts --libraries-spec=/home/mea/snap/flutter/common/flutter/bin/cache/dart-sdk/lib/libraries.json --packages=org-dartlang-app:///.dart_tool/package_config.json --multi-root-scheme=org-dartlang-app --multi-root=/tmp/scratch_spaceQOBQQM/ --native-null-assertions --sound-null-safety -oweb/worker.dart.js org-dartlang-app:///web/worker.dart

[INFO] 19.2s elapsed, 1878/1879 actions completed.
[INFO] 20.3s elapsed, 1878/1879 actions completed.
[INFO] 21.4s elapsed, 1878/1879 actions completed.
[INFO] 22.5s elapsed, 1878/1879 actions completed.
[INFO] build_web_compilers:entrypoint on web/worker.dart:Info: Compiling with sound null safety
Compiled 10,975,376 characters Dart to 1,045,446 characters JavaScript in 4.52 seconds
Dart file org-dartlang-app:///web/worker.dart compiled to JavaScript: web/worker.dart.js


[INFO] Running build completed, took 23.5s

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 340ms

[INFO] Succeeded after 23.9s with 1687 outputs (6661 actions)
$ flutter pub run drift_dev schema dump lib/database/database.dart drift_schemas/
Wrote to drift_schemas/drift_schema_v2.json
$ flutter pub run drift_dev schema generate drift_schemas/ test/generated_migrations/
Unhandled exception:
CheckedFromJsonException
Could not create `DriftOptions`.
There is a problem with "sql".
type 'DialectOptions' is not a subtype of type 'Map<dynamic, dynamic>' in type cast
#0      $checkedConvert (package:json_annotation/src/checked_helpers.dart:94:5)
#1      $checkedCreate._checkedConvert (package:json_annotation/src/checked_helpers.dart:31:7)
#2      _$DriftOptionsFromJson.<anonymous closure> (package:drift_dev/src/analyzer/options.g.dart:86:35)
#3      $checkedCreate.<anonymous closure> (package:json_annotation/src/checked_helpers.dart:36:22)
#4      $checkedNew (package:json_annotation/src/checked_helpers.dart:54:23)
#5      $checkedCreate (package:json_annotation/src/checked_helpers.dart:33:10)
#6      _$DriftOptionsFromJson (package:drift_dev/src/analyzer/options.g.dart:9:50)
#7      new DriftOptions.fromJson (package:drift_dev/src/analyzer/options.dart:171:46)
#8      GenerateUtilsCommand._writeSchemaFile (package:drift_dev/src/cli/commands/schema/generate_utils.dart:113:34)
#9      GenerateUtilsCommand.run (package:drift_dev/src/cli/commands/schema/generate_utils.dart:68:13)
<asynchronous suspension>
#10     CommandRunner.runCommand (package:args/command_runner.dart:209:13)
<asynchronous suspension>
#11     MoorCli.run (package:drift_dev/src/cli/cli.dart:65:5)
<asynchronous suspension>
#12     run (package:drift_dev/src/cli/cli.dart:19:12)
<asynchronous suspension>
pub finished with exit code 255
```

**Comment by simolus3** [comment]
Thanks for the report. I've fixed both issues in cf95ed6f056ea28cdb4444408336b882bd6c7db8.

**Comment by MateusAmin** [comment]
Absolute legend, Thanks! :)
